user.rb

```diff
@@ -1,16 +1,20 @@
 require "pry"
 
 class User
-  @@all_users = []
+  @all_users = []
   attr_accessor :first_name, :last_name, :email
 
   def self.all
-    @@all_users
+    @all_users
+  end
+
+  def self.all=(array_of_users)
+    @all_users = array_of_users
   end
 
   def self.create(first_name:, last_name:, email:)
     if User.is_email_in_use?(email)
-       return puts "Sorry this email is taken"
+       return puts "Sorry, the email #{email} is already in use"
```

Comment:
I think a custom exception would be better suited here. `puts` is usually only used in debugging.

What do you think of something like:

```suggestion
       raise ArgumentError "Sorry, the email #{email} is already in use"
```